packages/perseus/src/renderer.tsx

```diff
@@ -104,60 +106,6 @@ type SetWidgetPropsFn = (
     silent?: boolean,
 ) => void;
 
-type SerializedState = {
```

Comment:
This whole block was moved to `types.ts`. The `Widget` type also becomes an `interface` in that file.